### PR TITLE
Mp units and std chrono duration support with glaze

### PR DESCRIPTION
**Pull request by jbbjarnason**

### CODE REVIEW

File: libs/confman/inc/public/tfc/confman.hpp [hunk 1]
```diff
@@ -68,8 +68,18 @@ class config {
 
   /// \return storage_t as json string
   [[nodiscard]] auto string() const -> std::string { return glz::write_json(storage_.value()); }
+
+  /// TODO can we do this differently, jsonforms requires object as root element
+  template <typename some_type>
+  struct hack {
```
Comment (omarhogni) [inline libs/confman/inc/public/tfc/confman.hpp:74]:
```suggestion
  struct object_wrapper {
```

File: libs/confman/inc/public/tfc/confman.hpp [hunk 2]
```diff
@@ -68,8 +68,18 @@ class config {
 
   /// \return storage_t as json string
   [[nodiscard]] auto string() const -> std::string { return glz::write_json(storage_.value()); }
+
+  /// TODO can we do this differently, jsonforms requires object as root element
+  template <typename some_type>
```
Comment (omarhogni) [inline libs/confman/inc/public/tfc/confman.hpp:73]:
```suggestion
  template <typename to_be_wrapped_t>
```

File: libs/confman/inc/public/tfc/confman.hpp [hunk 3]
```diff
@@ -68,8 +68,18 @@ class config {
...
+  struct hack {
+    some_type placeholder{};
+    struct glaze {
+      static constexpr auto value{ glz::object("config", &hack::placeholder) };
+      static constexpr auto name{ glz::name_v<some_type> };
+    };
+  };
   /// \return storage_t json schema
-  [[nodiscard]] auto schema() const -> std::string { return glz::write_json_schema<config_storage_t>(); }
+  [[nodiscard]] auto schema() const -> std::string { return glz::write_json_schema<hack<config_storage_t>>(); }
```
Comment (omarhogni) [inline libs/confman/inc/public/tfc/confman.hpp:82]: same

**Review by omarhogni — APPROVED** [review]

File: libs/confman/inc/public/tfc/confman.hpp (hunk 1 above)
Comment (jbbjarnason) [reply libs/confman/inc/public/tfc/confman.hpp]: done

File: libs/confman/inc/public/tfc/confman.hpp (hunk 2 above)
Comment (jbbjarnason) [reply libs/confman/inc/public/tfc/confman.hpp]: done

File: libs/confman/inc/public/tfc/confman.hpp (hunk 3 above)
Comment (jbbjarnason) [reply libs/confman/inc/public/tfc/confman.hpp]: done

**Pull request merged**